qemu/tests/cfg/boot_with_max_queues.cfg

```diff
@@ -0,0 +1,27 @@
+- boot_with_max_queues:
+    type = boot_with_max_queues
+    image_snapshot = yes
+    queues = 256
+    start_vm = no
+    scp_count = 1
+    file_transfer_client = "scp"
+    filesize = 1
+    vt_ulimit_nofile = 10240
+    Linux:
+        msi_check_cmd = "lspci -vvv -s %s | grep MSI"
+    Windows:
+        no Win2008..sp2
+        cdroms += " virtio"
+
+        traceview_version = "win10"
+        Win7, Win8, Win8..1, Win2008r2, Win2012, Win2012r2:
+            traceview_version = "win8"
+  
+        i386:
+            arch_name = "x86"
+        x86_64:
+            arch_name = "amd64"
+        traceview_path_template = "%s:\traceview\${traceview_version}\${arch_name}\traceview.exe"

```

Comment:
here ${arch_name} can be replaced with %PROCESSOR_ARCHITECTURE%, if the guest arch is is386 will equal to x86, if the guest arch is x86_64 will equal to amd64,  thus will no need add 20-23 lines.
you can refer to    rng_bat.cfg:  read_rng_cmd  = "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"